_rules/aria-required-id-references-in6db8.md

```diff
@@ -0,0 +1,199 @@
+---
+id: in6db8
+name: ARIA required ID references exist
+rule_type: atomic
+description: |
+  This rule checks that every ID reference required by WAI-ARIA exists
+accessibility_requirements:
+  aria12:propcharacteristic_value:
+    title: ARIA 1.2, 6.2.4 Value (Characteristics of States and Properties)
+    forConformance: true
+    failed: not satisfied
+    passed: satisfied
+    inapplicable: satisfied
+  wcag20:1.3.1: # Info and Relationships (A)
+    secondary: true
+  wcag20:4.1.2: # Name, Role, Value (A)
+    secondary: true
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+acknowledgments:
+  authors:
+    - Wilco Fiers
+---
+
+## Applicability
+
+This rule applies to any HTML element with an `aria-controls` attribute for which one of the following is true:
+
+- <dfn>expanded combobox</dfn>: the element is a [semantic][] `combobox` with an `aria-expanded` [attribute value][] of `true`; or
+- <dfn>scrollbar</dfn>: the element is a [semantic][] `scrollbar`.
+
+## Expectation
+
+Each test target's `aria-controls` [attribute value][] is a space-separated list of one or more IDs. At least one of those IDs must match an `id` [attribute value][] in the same [document][document tree] or [shadow tree][].
```

Comment:
**Thought:** Do we want the rule to apply to the element or to the attribute?
The current [ARIA state or property has valid value](https://www.w3.org/WAI/standards-guidelines/act/rules/6a7281/proposed/) applies to the attribute.
```suggestion
This rule applies to any `aria-controls` attribute defined on an [HTML element][namespaced element] for which one of the following is true:

- <dfn>expanded combobox</dfn>: the element is a [semantic][] `combobox` with an `aria-expanded` [attribute value][] of `true`; or
- <dfn>scrollbar</dfn>: the element is a [semantic][] `scrollbar`.

## Expectation

Each test target's [attribute value][] is a space-separated list of one or more IDs. At least one of those IDs must match an `id` [attribute value][] in the same [document][document tree] or [shadow tree][].
```
(would also need to change a bit the examples descriptions)